package/MDAnalysis/analysis/diffusionmap.py

```diff
@@ -320,33 +300,9 @@ def run(self):
                                self._epsilon)
         # take negative exponent of scaled matrix to create Isotropic kernel
         self._kernel = np.exp(-self._scaled_matrix)
-        # define an anistropic diffusion term q
-        q_vector = np.zeros((self._nframes, ))
-        # weights should reflect the density of the points on the manifold
-        for i in range(self._nframes):
-            q_vector[i] = np.dot(self._kernel[i, :], self._weights_ker)
-
-        # Form a new kernel from the anisotropic diffusion term q
-        self._kernel /= (np.sqrt(q_vector[:, np.newaxis
-                         ].dot(q_vector[np.newaxis])))
-
-        # Weighted Graph Laplacian normalization on this new graph
-        d_vector = np.zeros((self._nframes, ))
-        for i in range(self._nframes):
-            d_vector[i] = np.dot(self._kernel[i, :], self._weights_ker)
-
-        for i in range(self._nframes):
-            self._kernel[i, :] = self._kernel[i, :] * self._weights_ker
-
-        # Define anisotropic transition by dividing kernel by this term
-        self._kernel /= np.sqrt(d_vector[:, np.newaxis].dot(d_vector[np.newaxis]))
-
-        # Apply timescaling
-        if self._timescale > 1:
-            self._kernel = np.linalg.matrix_power(self._kernel,
-                                                  self._timescale)
-
-        eigenvals, eigenvectors = np.linalg.eig(self._kernel)
+        D_inv = np.diag(1 / self._kernel.sum(1))
+        self._diff = np.dot(D_inv, self._kernel)
+        eigenvals, eigenvectors = np.linalg.eig(self._diff)
```

Comment:
I did and apparently I may have been wrong, the two give different results:
Eigenvalues using epsilon = 5, eigh:
```[ 1.02532315  0.63865029  0.50461156  0.46652167  0.43577238  0.33804695
  0.29255447  0.23455876  0.2047588   0.18443111  0.14899126  0.14097275
  0.12780406  0.11636266  0.11326329  0.11052915  0.10414109  0.10212337
  0.09150211  0.08408036]```

Eigenvalues using epsilon = 5, eig:
```
[ 1.          0.69843825  0.51574675  0.47646882  0.40558677  0.3209098
  0.25232349  0.21522759  0.20871485  0.17725692  0.14328863  0.13184095
  0.11912811  0.11392278  0.10615283  0.10202534  0.10149993  0.09744823
  0.08432603  0.07256051]
```